pkg/gatherers/workloads/gather_workloads_info.go

```diff
@@ -496,11 +525,26 @@ func calculateWorkloadContainerShapes(
 			firstArg = shortHash
 		}
 
+		var runtimeInfo workloadRuntimeInfoContainer
```

Comment:
when workload data is gathered, we go through the `namespace / pod / containerID / workloadRuntimeInfos` map and populate the shapes with them